datasets/flwr_datasets/partitioner/size_partitioner_test.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2023 Flower Labs GmbH. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""SizePartitioner tests."""
+import unittest
```

Comment:
We usually have two blank lines between docstring and first import, let's keep this consistent:
```suggestion
"""SizePartitioner tests."""


import unittest
```